tfe/data_source_workspace.go

```diff
@@ -193,6 +185,27 @@ func dataSourceTFEWorkspaceRead(d *schema.ResourceData, meta interface{}) error
 	d.Set("trigger_prefixes", workspace.TriggerPrefixes)
 	d.Set("working_directory", workspace.WorkingDirectory)
 
+	// Set remote_state_consumer_ids if global_remote_state is false
+	globalRemoteState := workspace.GlobalRemoteState
+	if globalRemoteState {
+		if err := d.Set("remote_state_consumer_ids", []string{}); err != nil {
+			return err
+		}
+	} else {
+		compatibleGlobalRemoteState, remoteStateConsumerIDs, err := readWorkspaceStateConsumers(workspace.ID, tfeClient)
```

Comment:
The name `compatibleGlobalRemoteState` didn't really tell me what this value is, and I had to stare at `readWorkspaceStateConsumers` for a long time to understand it.

Ignoring the fact that the function should probably have godoc on it explaining the return value (😅 That's my bad), how about something like `legacyGlobalState` or `oldTFE` or something to indicate that `true` == "This is an old version of TFE and for all points and purposes uses global state" ?